internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -224,7 +220,7 @@ func (r *BundleReconciler) ensureUnpackPod(ctx context.Context, bundle *rukpakv1
 			pod.Spec.Containers = make([]corev1.Container, 1)
 		}
 		pod.Spec.Containers[0].Name = bundleUnpackContainerName
-		pod.Spec.Containers[0].Image = bundle.Spec.Image
+		pod.Spec.Containers[0].Image = bundle.Spec.Source.Image.Ref
```

Comment:
Yep - it seems like we may need to house a switch/case statement in the Reconcile method that basically builds up a concrete Unpacker type for unpacking/sourcing different Bundle source types. Maybe this can be done as a follow-up?